### DISCUSSION TITLE
basic usage dialog in UI

### DISCUSSION THREAD

**Discussion by edgcumt** [arkypita/LaserGRBL#2402] [Q&A] [answered]
searched allsorts for how to do this with no luck.

I just want to hide the "basic usage" and "keyboard shortcuts" dialog the sits in the bottom right corner of the preview area. is this something i'm overlooking in the settings or maybe a something that's coming in next update? 

i'm using version 7.12.0

**Comment by arkypita** [comment, marked as answer]
As soon as you load your job they disappare, why need to hide?

**Reply by edgcumt** [reply]
Oh, nice! I should have just tried it out myself to see how it worked. None of the tutorials I watched showed that, so I assumed it was something you could adjust in the settings.

Thanks for your help
